openquakeplatform/openquakeplatform/test/isc_test.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+import time
+import sys
+import unittest
+
+# from IPython.core import ultratb
+# sys.excepthook = ultratb.FormattedTB(mode='Verbose',
+#      color_scheme='Linux', call_pdb=1)
+
+from openquakeplatform.test import pla
+from openquakeplatform.test.utils import *
+
+
+class IscTest(unittest.TestCase):
+    def isc_test(self):
+        pla.navigate('explore')
+
+        #<li>
+        #<a href="/maps/23">
+        enter_button = pla.xpath_finduniq(
+            "//li/a[@href='/maps/23' and normalize-space(text()) = 'Global "
+            "Instrumental Earthquake Catalogue (1900 - 2009)']")
+        enter_button.click()
+        pla.wait_new_page(enter_button, '/maps/23')
+
```

Comment:
I do not like much that we reuse the variable `enter_button` to refer to different buttons. Can't we use different names so that it is clear to which button are we referring?